api/IdleDetector.json

```diff
@@ -4,6 +4,9 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/IdleDetector",
         "spec_url": "https://wicg.github.io/idle-detection/#api-idledetector",
+        "tags": [
+          "webfeature:idle-detection"
```

Comment:
OK, after sitting with this a bit, this is the one little thing I'm not excited about.

Narrowly, I think we should go with `web-features` as the name space, since it directly relates to that project. I'm pretty certain that if they're close but not the same, I'll routinely type `webfeatures` or `web-feature` or some other wrong variation. Please don't make me think. 😅

More broadly, I can write about conventions around plurals in BCD if you want me to, but my narrow complaint is probably my strongest and most practical note.

Reply:
Fun fact: This PR makes this mistake and sometimes uses "webfeature" and sometimes "webfeatures". I will therefore accept your narrow complaint and go with "web-features" consistently 😄 

(I thought the singular form looks better but lets talk about plurals vs singulars when we can have a coffee together or so :)